### DISCUSSION TITLE
2010/10/23/%E6%9C%80%E8%BF%91%E5%8F%AF%E8%83%BD%E4%B8%8D%E4%BC%9A%E6%9B%B4%E6%96%B0blog%E4%BA%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#298] [Announcements]
# 2010/10/23/%E6%9C%80%E8%BF%91%E5%8F%AF%E8%83%BD%E4%B8%8D%E4%BC%9A%E6%9B%B4%E6%96%B0blog%E4%BA%86

等Lives搬好了家再继续写吧

https://blog.qizhen.xyz/2010/10/23/%E6%9C%80%E8%BF%91%E5%8F%AF%E8%83%BD%E4%B8%8D%E4%BC%9A%E6%9B%B4%E6%96%B0blog%E4%BA%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

科学玩家
2010年11月6日 01:11 编辑
现在搬家到wordpress了，估计快更新了吧？:)

回复
yoyo
2010年11月4日 21:24 编辑
嫂子肚子不大哦 快生了吧

回复
